src/main/java/com/github/wovnio/wovnjava/PathUrlLanguagePatternHandler.java

```diff
@@ -34,6 +40,15 @@ public boolean canInterceptUrl(String url) {
         return this.matchSitePrefixPathPattern.matcher(url).lookingAt();
     }
 
+    /*
+     * Redirect to same URL without language code if the language code
+     * found in the URL path is for default language
+     */
+    public boolean shouldRedirectToDefaultLang(String url) {
```

Comment:
Don't see any tests for this 😭 
`shouldRedirectExplicitDefaultLangUrl` would be my choice of name. "Explicit default language" I felt describes this scenario a bit more.

Would potentially also remove this from the abstract base class and make it mandatory. In wovn.cs it is done for all patterns other than custom_domain, like subdomain should redirect en.domain.com to domain.com too

Reply:
Thanks for the feedback! I'll try to adjust this accordingly.

Request handling seems to never have been tested very well, particularly for redirections and forwarding. The challenge is in mocking our environment. I'm still undecided in how to best add more robus testing, so for now I'm ok with moving forwards step by step, even if it's not perfect. That said, I'll look into adding something for this PR as well.

Reply:
I changed the variable name and added unit tests for it. Thanks for the heads up on that. I'd like redirection to be tested better at a higher level, but that will be a separate effort.

For the default implementation of the function, I'd prefer to leave it like this for now since only path pattern has a custom implementation. We can discuss whether subdomain and query should also have default lang redirections, but I would prefer to not bundle such a change here.